core/src/main/java/com/exadel/aem/toolkit/core/authoring/models/Slider.java

```diff
@@ -55,6 +65,34 @@ public class Slider extends BaseModel {
     @ValueMapValue
     private boolean filled;
 
+    @ValueMapValue
+    private String startValue;
+
+    @ValueMapValue
+    private String endValue;
+
+    @ValueMapValue
+    private boolean ranged;
+
+    @ValueMapValue
+    private SliderItem[] items;
+
+    @ValueMapValue
+    private Map<String, String> itemsMap;
+
+    @PostConstruct
+    private void init() {
+        itemsMap = new HashMap<>();
+        Resource resourceChild = resource.getChild(CoreConstants.NN_ITEMS);
+        if (resourceChild != null) {
+            Iterator<Resource> resources = resourceChild.listChildren();
+            while (resources.hasNext()) {
+                Resource it = resources.next();
+                itemsMap.put(it.getValueMap().get(CoreConstants.PN_VALUE, String.class), it.getValueMap().get(CoreConstants.PN_TEXT, String.class));
```

Comment:
(cosmetic) This line is too long. Consider splitting it like 
```
itemsMap.put(
    it.getValueMap(),,,,
    item.getValueMap()... );
```
But even before that, think of the following: `it.getValueMap().get(CoreConstants.PN_VALUE, String.class)` can return null. This is not improbable. Repository can be corrupted, or someon put a hand on data consistency, etc. Does it make sense to inser null values?